datastore/src/main/java/org/jboss/pnc/datastore/DefaultDatastore.java

```diff
@@ -148,28 +151,40 @@ public BuildRecord storeCompletedBuild(BuildRecord.Builder buildRecordBuilder) {
      */
     private Set<Artifact> saveArtifacts(Collection<Artifact> artifacts) {
         logger.debug("Saving {} artifacts.", artifacts.size());
+
         Set<Artifact> savedArtifacts = new HashSet<>();
         for (Artifact artifact : artifacts) {
             TargetRepository targetRepository = saveTargetRepository(artifact.getTargetRepository());
             artifact.setTargetRepository(targetRepository);
+
             Artifact artifactFromDb;
             if (TargetRepository.Type.GENERIC_PROXY.equals(artifact.getTargetRepository().getRepositoryType())) {
                 artifactFromDb = saveHttpArtifacts(artifact);
             } else {
                 artifactFromDb = saveRepositoryArtifacts(artifact);
             }
+
             savedArtifacts.add(artifactFromDb);
         }
+
         logger.debug("Artifacts saved: {}.", artifacts);
         return savedArtifacts;
     }
 
     private TargetRepository saveTargetRepository(TargetRepository targetRepository) {
+        logger.trace("Saving target repository {}.", targetRepository);
         TargetRepository targetRepositoryFromDb = targetRepositoryRepository
                 .queryByIdentifierAndPath(targetRepository.getIdentifier(), targetRepository.getRepositoryPath());
+
         if (targetRepositoryFromDb == null) {
+            logger.trace("Target repository is not in DB. Saving target repository {}.", targetRepository);
             targetRepositoryFromDb = targetRepositoryRepository.save(targetRepository);
+            logger.trace("Target repository saved {}.", targetRepositoryFromDb);
+        }
+        else {
```

Comment:
Follow the code style "} else {".

Reply:
OK, changing it.

Obviously we are used to different code style. We need the automatic formatters :-(. I realized we have the repo with code-style definition. Is it up to date. If so, I'd import it to Idea and use it for all changes. https://github.com/project-ncl/ide-config